include/gatekeeper_net.h

```diff
@@ -352,6 +352,9 @@ struct net_config {
 	 */
 	bool                 *numa_used;
 
+	/* The lcores used in the host. */
+	bool                 lcores_used[RTE_MAX_LCORE];
```

Comment:
Once field `gatekeeper_pktmbuf_pool` is removed, field `lcores_used` is no longer needed as well.